integration/environment/environment.go

```diff
@@ -246,15 +248,28 @@ func (e *Environment) setupCFOperator(namespace string) (err error) {
 
 	loggerPath := helper.LogfilePath(fmt.Sprintf("cf-operator-tests-%d.log", e.ID))
 	e.ObservedLogs, e.Log = helper.NewTestLoggerWithPath(loggerPath)
+	crlog.SetLogger(zapr.NewLogger(e.Log.Desugar()))
```

Comment:
I think we should use the same prefix like here:

https://github.com/cloudfoundry-incubator/cf-operator/blob/master/cmd/internal/root.go#L162

Or are the cr log messages already prefixed with 'kubebuilder' all the time?

Reply:
All messages coming from here are already prefixed with `controller-runtime.*`.
Example:
```
2019-07-29T14:27:56.698+0300    INFO    controller-runtime.controller   Stopping workers        {"controller": "ext-statefulset-controller"}
2019-07-29T14:27:56.698+0300    INFO    controller-runtime.controller   Stopping workers        {"controller": "ext-job-job-controller"}
```

Reply:
@manno should I remove the `cr` prefix from root?

Reply:
Yes, if it's not too much trouble. Thanks for cleaning this up!